runtime/opt/taupage/init.d/01-register-scalyr-agent.py

```diff
@@ -0,0 +1,214 @@
+            scalyr_api_key = subprocess.check_output(['python3',
+                                                      '/opt/taupage/bin/decrypt-kms.py',
+                                                      scalyr_api_key]).decode('UTF-8').strip()
+        except Exception:
+            logger.error('Failed to run /opt/taupage/bin/decrypt-kms.py')
+            raise SystemExit(1)
+    if scalyr_api_key == "Invalid KMS key.":
+        logger.error('Failed to decrypt KMS Key')
+        raise SystemExit(1)
+    return scalyr_api_key
+
+
+def parse_string(value):
+    try:
+        result = ast.literal_eval(value)
+    except Exception:
+        logger.warning('String \"{!s}\" could not be parsed, will be ignored!'.format(value))
+        result = None
+    return result
+
+
+def create_config_skeleton():
+    instance_data = boto.utils.get_instance_identity()['document']
+    scalyr_config = {
+        'api_key': decrypt_scalyr_key(),
+        'scalyr_server': 'https://upload.eu.scalyr.com',
+        'compressionType': 'bz2',
+        'debug_init': True,
+        'max_log_offset_size': 30000000,
+        'read_page_size': 131072,
+        'max_line_size': 49900,
+        'implicit_agent_process_metrics_monitor': False,
+        'implicit_metric_monitor': False,
+        'server_attributes': {
+            'application_id': main_config.get('application_id'),
+            "serverHost": main_config.get('application_id'),
+            'application_version': main_config.get('application_version'),
+            'stack': main_config.get('notify_cfn', {}).get('stack'),
+            'source': main_config.get('source'),
+            'image': main_config.get('source').split(':', 1)[0],
+            'aws_region': instance_data.get('region'),
+            'aws_account': instance_data.get('accountId'),
+            'aws_ec2_hostname': boto.utils.get_instance_metadata()['local-hostname'],
+            'aws_ec2_instance_id': instance_data.get('instanceId')
+        },
+        'logs': [],
+        'monitors': []
+    }
+    return scalyr_config
+
+
+def create_log_item(logfile, parser, do_jwt_redaction, sampling):
+    item = {
+        'path': logfile,
+        'copy_from_start': True,
+        'attributes': {
+            'parser': parser
+        }
+    }
+    if do_jwt_redaction:
+        item['redaction_rules'] = jwt_redaction
+
+    if sampling:
+        item['sampling_rules'] = parse_string(sampling)
+
+    return item
+
+
+if __name__ == '__main__':
+    config = create_config_skeleton()
+    if ship_application_log:
+        config['logs'].append(
+            create_log_item(
+                application_log_path,
+                application_log_parser,
+                True,
+                application_log_sampling
+            )
+        )
+
+    if (ship_custom_log and mount_custom_log):
+        config['logs'].append(
+            create_log_item(
+                custom_log_path,
+                custom_log_parser,
+                True,
```

Comment:
As far as I can see, we did not do jwt redaction before for custom logs.
Was this feature intended?